### PR TITLE
Change editors specification

### PR DESCRIPTION
**Pull request by maartenweyns**
I have updated the editors section on the systems page. Jetbrains editors and VS code had all been updated to newer versions

### CODE REVIEW

File: content/systems.md [hunk 1]
```diff
@@ -30,27 +30,27 @@ The software configuration will consist of the following:
     - geany
     - kate
 - IDEs:
-    - IntelliJ Community
-        - Version: 2021.3.2
-        - Workspace is already set up
+    - Idea Community
+        - Version: 2022.3
+        - Workspace is already set up but needs to be opened from the welcome screen
```
Comment (mpsijm) [inline content/systems.md:35]: If this is not the default flow when opening IntelliJ, we shouldn't forget to mention this during the opening presentation. The default flow I have in mind, is:
- You double-click the IntelliJ icon
- The "Welcome" popup window appears
- You need to select the already-set-up workspace

Also, given that you did not write this for CLion and PyCharm, does that mean that those editor _do_ directly load into the already-set-up workspace on launch?

Comment (maartenweyns) [reply content/systems.md:35]: Yes, indeed. CLion and PyCharm open the workspace directly.

**Comment by mpsijm** [comment]
@maartenweyns I see you added Robert as reviewer, would you like him to double-check the versions or can this PR be merged after resolving my question above? :slightly_smiling_face:

File: content/systems.md (hunk 1 above)
Comment (maartenweyns) [reply content/systems.md:35]: But it is easy to see what to click, the workspace is just listed in the welcome screen

**Review by mpsijm — APPROVED** [review]

File: content/systems.md (hunk 1 above)
Comment (mpsijm) [reply content/systems.md:35]: Exactly, just what I expected :slightly_smiling_face: Thanks for clarifying! :smile:

File: content/systems.md [hunk 2]
```diff
@@ -30,27 +30,27 @@ The software configuration will consist of the following:
     - geany
     - kate
 - IDEs:
-    - IntelliJ Community
-        - Version: 2021.3.2
-        - Workspace is already set up
+    - Idea Community
```
Comment (verwoerd) [inline content/systems.md:33]: The official name is `IntelliJ IDEA` (Note the capitalization too)

**Review by verwoerd — APPROVED** [review]

**Comment by maartenweyns** [comment]
It can be merged :)

**Pull request merged**